src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -112,6 +112,40 @@ protected bool ShouldProcessServiceOperation(ServiceController service)
             WriteError(new ErrorRecord(exception, errorId, category, targetObject));
         }
 
+        internal void SetServiceSecurityDescriptor(
+            ServiceController service,
+            string securityDescriptorSddl,
+            NakedWin32Handle hService)
+        {
+            var rawSecurityDescriptor = new RawSecurityDescriptor(securityDescriptorSddl);
+            RawAcl rawDiscretionaryAcl  = rawSecurityDescriptor.DiscretionaryAcl ;
+            var  discretionaryAcl   = new DiscretionaryAcl (false, false, rawDiscretionaryAcl);
```

Comment:
Extra spaces in two lines above.

Reply:
Thanks, why didn't CodeFactor catch this ?

Reply:
Sometimes it is not predictable :-)